modules/select2_publish/select2_publish.info.yml

```diff
@@ -0,0 +1,8 @@
+name: Select2 Publish
+type: module
+description: Indicates the status of referenced entities.
+package: User interface
+core: '8.x'
+dependencies:
+  - select2:select2
+  - form_options_attributes:form_options_attributes
```

Comment:
Is there some why that we have correct module dependencies for composer too?

Reply:
Not really, but never the less I added form_options_attributes:form_options_attributes to require-dev to have it at least downloaded in our tests